Dockerfile

```diff
@@ -13,3 +14,7 @@ RUN mkdir /app/migrations
 COPY --from=builder /app/bin/dp210goapp /app
 COPY ./migrations /app/migrations
 COPY ./config.json /app
+EXPOSE 8000
+RUN apt update
+RUN apt install -y curl
```

Comment:
do it like this:

1. RUN apt update
2. RUN apt install -y curl
3. COPY ./migrations /app/migrations
4. COPY ./config.json /app
5. COPY --from=builder /app/bin/dp210goapp /app

from [docker docs](https://docs.docker.com/engine/reference/commandline/images/) 
> Docker images have intermediate layers that increase reusability, decrease disk usage, and speed up docker build by allowing each step to be cached. These intermediate layers are not shown by default.
it meens that every line is cached as a container above previous, if only last line will be executed most of times, our container will start much faster